mmv1/products/dialogflowcx/api.yaml

```diff
@@ -17,7 +17,7 @@ display_name: Dialogflow CX
 versions:
   - !ruby/object:Api::Product::Version
     name: ga
-    base_url: https://{{location}}-dialogflow.googleapis.com/v3/
+    base_url: https://dialogflow.googleapis.com/v3/
```

Comment:
curious, why was `location` added in the first place? did the API change?

Reply:
yes it seems like API changed and now isn't regional